identityClient/api.go

```diff
@@ -60,6 +60,14 @@ type RealmInfo struct {
 	Domain                string    `json:"domain"`
 }
 
+// PrivateRealmInfo represents the private information about a TozID realm
+type PrivateRealmInfo struct {
```

Comment:
`PrivateRealmInfoRequest`

Reply:
never mind...

Reply:
but you need to figure out what to do with name, either the method needs to fill it in or pull it from the response struct